source/reference/api/start.txt

```diff
@@ -132,6 +132,9 @@ Request Body Parameters
        original source cluster blocks writes and the destination cluster
        accepts writes.
 
+       :gold:`IMPORTANT:` If you are migrating from a pre-6.0 source cluster, 
```

Comment:
[optional suggestion]

You might consider making this an include given that it shows up twice on this page.

Reply:
Ah the second instance is supposed to be say "you cannot set reversible. Updating.